frontend/plugins/pos_ui/src/modules/create-pos/components/sync/sync.tsx

```
@@ -11,9 +11,16 @@ interface SyncCardFormProps {
   posDetail?: IPosDetail;
   isReadOnly?: boolean;
 }
+import { IPosDetail } from '~/modules/pos-detail/types/IPos';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'IPosDetail'. Consider to delete it or rename it.

'IPosDetail' is defined here:


(lint/suspicious/noRedeclare)